src/helpers/request/request-helper.js

```diff
@@ -54,29 +54,49 @@ const requestTranscriptQuery = (parent_id) => `query {
 }`;
 
 export const fetchRequestTranscript = (requestId, persona) => {
-  const fetchHeaders = { 'x-rh-persona': persona ? persona : 'approval/requester' };
+  const fetchHeaders = (persona && persona !== 'approval/approver') ? { 'x-rh-persona': persona } : { 'x-rh-persona': 'approval/requester' };
```

Comment:
We could also replace these with constants

Reply:
@Hyperkid123 Missed these - fixed now, thanks